### DISCUSSION TITLE
왼쪽 사이드바를 고정하고 싶습니다

### DISCUSSION THREAD

**Discussion by recode-sm** [pronist/hELLO#156] [Q&A] [answered]
왼쪽 사이드바를 고정하고 싶습니다 어떻게 수정 해야할까요?

**Comment by pronist** [comment, marked as answer]
@recode-sm 현재 가로사이즈 `1400px` 이상에서는 기본값으로 사이드바가 고정으로 되어 있을텐데, CSS 를 수정하면 되긴 하지만, `1400px` 이하에서도 사이드바가 고정되면 모바일 환경에서 사용자 경험이 굉장히 나빠질겁니다. 정말 하고 싶으시다면, `body[data-sidebar-default]` 가 쓰인 CSS 부분을 지우는 방향으로 수정해보시기 바랍니다. :)

**Reply by recode-sm** [reply]
상우님의 티스토리와 같이 왼쪽에 고정하고 싶으면 body[data-sidebar-default]를 제거하면 될까요? 빠른 답변 감사합니다

**Reply by pronist** [reply]
@recode-sm 제 블로그는 아무것도 건드리지 않은 것입니다~

**Reply by recode-sm** [reply]
저는 스킨을 다운받아서 적용했는데 상단의 메뉴에 티스토리 아이콘을 눌러야지 사이드바 메뉴가 나오고 메뉴바가 아닌 영역에 클릭하면
나왔던 메뉴바도 사라져 버립니다

**Reply by pronist** [reply]
@recode-sm 블로그 링크 남겨주세요~

**Reply by recode-sm** [reply]
https://code-mo.tistory.com/ 블로그 주소입니다

**Reply by recode-sm** [reply]
상우님의 사이드바 on off를 이제 이해한거 같습니다 스킨 편집에서 설정을 해주니 문제가 해결되었습니다 감사합니다

**Reply by kylo-dev** [reply]
저도 왼쪽 사이드바가 아이콘을 클릭해야 나오는데, 고정으로 두고 싶은데 어떻게 해결했나요?

**Reply by damon-911** [reply]
저도 궁금합니다

**Comment by jiinyoo** [comment]
나중에 어떤 오류가 생길지 모르겠으나 저는 css파일의 2004번째 줄의 픽셀을 1500PX로 올리고
@media (min-width: 1500px) 
2114번째 줄을 1000px로 줄이니까 사이드바가 고정 되었습니다.
@media (min-width: 1000px)
